### PR TITLE
implement upload.fromUrl 

### PR DESCRIPTION
**Pull request by evaletolab**
This is a simple but efficient implementation of upload.fromUrl.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,21 +5,31 @@ Node module for uploadcare.com that allows API interaction as well as local file
...
-    //Upload
+    //
+    //Upload from file
     uploadcare.file.upload(fs.createReadStream(path), function(err,res){
         //Res should contain returned file ID
         console.log(err,res);
+    });
+    
+    //
+    //Upload from file
```
Comment (dmitry-mukhin) [inline README.md]: from url?

Comment (evaletolab) [inline README.md]: yep, you right, sorry ;-)

**Comment by evaletolab** [comment]
what is the status on this pull request?

**Comment by oleduc** [comment]
@RexMorgan ?

**Comment by RexMorgan** [comment]
Thanks, sorry for the delay.

**Comment by RexMorgan** [comment]
@evaletolab @oleduc 

This is in 0.3.0 that was released to npmjs.com a couple of hours ago.

Thanks again!

**Comment by evaletolab** [comment]
Nice, thanks !

On Mon, Jun 15, 2015 at 9:32 PM, Rex Morgan <notifications@github.com>
wrote:

> @evaletolab <https://github.com/evaletolab> @oleduc
> <https://github.com/oleduc>
>
> This is in 0.3.0 that was released to npmjs.com a couple of hours ago.
>
> Thanks again!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/RexMorgan/uploadcare-node/pull/6#issuecomment-112181763>
> .
>



-- 
https://github.com/evaletolab
http://evaletolab.ch

**Pull request merged**